gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -304,6 +304,8 @@ user:
   mfaVerifyAttempt:
     email:
       #subject: Multiple verification attempt detected
+  webAuthn:
+    latestCredentialsCount: 20 # number of credentials sent back to perform passwordless authentication. Keep in mind, Firefox handles max 20
```

Comment:
Not a big fan of this variable name. I would like to follow the W3C convention and name it : `maxAllowCredentials`.

Reply:
I was never good at names.